src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -233,9 +233,9 @@ public Videobridge()
      * @return a new <tt>Conference</tt> instance with an ID unique to the
      * <tt>Conference</tt> instances listed by this <tt>Videobridge</tt>
      */
-    public Conference createConference(String focus, String name)
+    public Conference createConference(String focus, String name, String gid)
```

Comment:
If `gid` can indeed be null, changing the method signature here breaks the API (I know it did for me, which is what triggered me to write this in the first place). Perhaps adding overloaded methods (where `gid` is null) would be good (which would restore backwards compatibility). Also, the new argument is not defined in javadoc.